documentation/assemblies/assembly-configure-address-spaces-addresses-cli-kube.adoc

```diff
@@ -0,0 +1,21 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+:parent-context: {context}
+:context: kube
+:cmdcli: kube
```

Comment:
should be `kubectl`

Reply:
@lulf, thanks. Also fixing assembly-monitoring-kube.adoc in this commit (since that's where I copied this from)